lib/srv/discovery/fetchers/db/azure.go

```diff
@@ -150,6 +150,20 @@ func (f *azureFetcher[DBType, ListClient]) FetcherType() string {
 	return f.cfg.Type
 }
 
+// IntegrationName returns the integration name.
+// There are no integrations supporting Azure Discovery fetchers, so this returns an empty string.
+func (f *azureFetcher[DBType, ListClient]) IntegrationName() string {
+	return ""
+}
+
+// DiscoveryConfigName is the name of the discovery config which originated the resource.
+// It is used to report stats for a given discovery config.
+// Might be empty when the fetcher is using static matchers:
+// ie teleport.yaml/discovery_service.<cloud>.<matcher>
+func (f *azureFetcher[DBType, ListClient]) DiscoveryConfigName() string {
+	return ""
```

Comment:
same question as above, should `DiscoveryConfigName` be set when there is no integration?